lib/internal/Magento/Framework/Module/Test/Unit/ModuleList/LoaderTest.php

```diff
@@ -54,7 +55,9 @@ protected function setUp()
         $this->parser->expects($this->once())->method('initErrorHandler');
         $this->registry = $this->createMock(\Magento\Framework\Component\ComponentRegistrarInterface::class);
         $this->driver = $this->createMock(\Magento\Framework\Filesystem\DriverInterface::class);
-        $this->loader = new Loader($this->converter, $this->parser, $this->registry, $this->driver);
+
+        $sorter = new Sorter();
```

Comment:
We cannot test **two** classes in **unit** test. Create a new test specifically for `SequenceSorter` and remove anything unrelated from this one.

UPD: great, you already created such test 👍 Just mock `SequenceSorter` here then.